Extension/src/LanguageServer/configurations.ts

```diff
@@ -308,7 +308,7 @@ export class CppProperties {
         if (isUnset(settings.defaultIncludePath)) {
             configuration.includePath = [rootFolder].concat(this.vcpkgIncludes);
         } else {
-            configuration.includePath = [rootFolder].concat(this.vcpkgIncludes).concat([defaultFolder]);
+            configuration.includePath = [defaultFolder].concat(this.vcpkgIncludes);
```

Comment:
I don't think we want to add the `vcpkgIncludes` here either because that's not the behavior they get when they don't have `c_cpp_properties.json`.  Just copy the `defaultFolder` into the includePath (`slice(0)`, I think)

Reply:
Yes

Reply:
the `defaultFolder `is a string: 
`const defaultFolder: string = "${default}";`
Will this be good? `configuration.includePath = [defaultFolder]`, or should I find the default folders defined in `c_cpp.default.includePath` and add them here (using` slice(0)`)?

Reply:
defaultFolder seems good